src/UI/Components/Diagram/actions.ts

```diff
@@ -259,7 +301,7 @@ export function appendInstance(
  * @param {EmbeddedEntity} embeddedEntity that we want to display
  * @param {InstanceAttributeModel} entityAttributes - attributes of given entity
  * @param {string | null} embeddedTo - id of the entity/shape in which this shape is embedded
- * @param {string} holderType - name of the entity type in which is embedded
+ * @param {string} holderName - name of the entity type in which is embedded
```

Comment:
name or type are not the same for me, also, maybe parentEntityType or parentEntityName might be clearer? 
Either the description should be 
```suggestion
 * @param {string} holderName - name of the entity to which it is embedded/connected
```
```suggestion
 * @param {string} holderType - Type of the entity to which it is embedded/connected
```